trymito.io/pages/teams.tsx

```diff
@@ -0,0 +1,232 @@
+import type { NextPage } from 'next'
+import Head from 'next/head'
+import Image from 'next/image'
+import Footer from '../components/Footer/Footer';
+import Header from '../components/Header/Header';
+import securityStyles from '../styles/Security.module.css'
+import pageStyles from '../styles/Page.module.css'
+import titleStyles from '../styles/Title.module.css'
+import homeStyles from '../styles/Home.module.css'
+import teamsStyles from '../styles/Teams.module.css'
+import CTAButtons from '../components/CTAButtons/CTAButtons';
+import ContactCTACard from '../components/CTACards/ContactCTACard';
+
+// Import Icons
+import JupyterIcon from '../public/icon-squares/JupyterIcon.svg'
+import SupportIcon from '../public/icon-squares/SupportIcon.svg'
+import DocsIcon from '../public/icon-squares/DocsIcon.svg'
+import PrivateIcon from '../public/icon-squares/PrivateIcon.svg'
+import Link from 'next/link';
+
+
+const Teams: NextPage = () => {
+
+  return (
+    <>
+      <Head>
+        <title>Mito | Teams </title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+      </Head>
+      
+      <Header/>
+    
+      <div className={pageStyles.container}>
+
+        <main className={pageStyles.main}>
+
+            <section className={titleStyles.title_card + ' ' + titleStyles.grid_card}>
+                <h1 className={titleStyles.title}>
+                    Embrace Python by leveraging your team’s Excel mastery
```

Comment:
These tag lines are always so hard! I don't feel like this is super clear about what it allows Mito to do. Here are some ideas:

- Transition to Python by leveraging your team's spreadsheet mastery
- The fastest way to transition your team's spreadsheet users to Python
- Make spreadsheet users Python Pros in minutes
- The only way to go from Spreadsheet Sucker to Python Pro

Pretty much: embrace and mastery are both kinda meaningless. Also, even though we are explicitly talking to the team leader, saying "your team" might be a bit much...